.github/workflows/run_fvt_tests.yaml

```diff
@@ -109,10 +109,10 @@ jobs:
         for file in `ls *.jtl`
         do
           if [ ! -z "$(cat $file| grep '<failure>' | awk -F '>' '{print $2}' | awk -F '<' '{print $1}' | grep true)" ] ||
-           [ "$(xmllint --format --xpath '/testResults/sample/@rc' $file | sed -r 's/ /\n/g;' | sort -u | grep -E 'rc=\"[45][0-9][0-9]\"|rc=\"\"')" != "" ]; then
-            echo -e "---------------------------------------------\n"
-            echo "FVT tests error for $file"
-            exit 1
+            [ "$(xmllint --format --xpath '/testResults/sample/@rc' $file | sed -r 's/ /\n/g;' | sort -u | grep -E 'rc=\"[45][0-9][0-9]\"|rc=\"\"')" != "" ]; then
```

Comment:
Why adding spaces

Reply:
Just format, because the original once space does not match the other indentations.